db/queries/exec_ssh_key.sql

```diff
@@ -0,0 +1,19 @@
+-- name: ExecSSHKeyInsert :exec
+INSERT INTO unweave.exec_ssh_key (exec_id, ssh_key_id)
+VALUES ($1, $2);
+
+-- name: ExecSSHKeyGet :one
+SELECT *
+FROM unweave.exec_ssh_key
+WHERE exec_id = $1
+  AND ssh_key_id = $2;
+
+-- name: ExecSSHKeysByExecIDGet :many
```

Comment:
This should be names `ExecSSHKeyGetByExecID` to comply with the current convention